### PR TITLE
sftp: add navigation and transfer examples

### PR DESCRIPTION
**Pull request by slash3b**
Just added a few commands to sftp page.

### CODE REVIEW

File: pages/common/sftp.md [hunk 1]
```diff
@@ -11,6 +11,10 @@
 
 `sftp -P {{remote_port}} {{remote_host}}`
 
+- Navigate through the LOCAL system file hierarchy:
+
+`lpwd, lls, lcd`
```
Comment (85pando) [inline pages/common/sftp.md]: While I am totally for including those three commands, it should probably rather be split in three separate commands with an explanation each

**Comment by 85pando** [comment]
I know this was in here before you edited it, but I just want to bring this up.

The commands "Copy local file to remote" and "Copy remote file to local" don't work for me and I have never seen this with `sftp` before (and I don't find it in the manpages). This seems to me like `scp`-syntax. Seing as this was added via a "Rework scp" PR it might come from here.
I think they should both be removed. This would also make more space for `cd`, `ls`, `cd` which are also missing from the page (maybe also `du [-h]`).

File: pages/common/sftp.md (hunk 1 above)
Comment (slash3b) [inline pages/common/sftp.md]: I think these commands are pretty easy to catch up with. User who intended to use `sftp` will guess that `lpwd` stands for _local present working directory_.  I'm just trying to keep thing concise. Please let me know if I have to split them.

**Comment by slash3b** [comment]
@85pando thanks. I've checked `get` and `put` commands on Ubuntu 14.04 and Fedora 23, please take a look at this screenshot ([sftp example  - connecting from Ubuntu 14.04 to Fedora 23 ](http://i.imgur.com/pB5JapK.png) ). On the left side there is man page for `sftp` and on the right side I tested the commands.

**Comment by 85pando** [comment]
I think you mistook me. I'm talking about those two:

* Copy a local file to a remote host:

`sftp {{local_file}} {{remote_host}}:{{/path/remote_file}}`

* Copy a remote file to the local host:

`sftp {{remote_host}}:{{/path/remote_file}} {{local_file}}`

**Comment by slash3b** [comment]
@85pando oh yes, I did misunderstand you, sorry  : )
Could you please review the changes I made ?

**Comment by 85pando** [comment]
Looking good. You might want to also add a "navigate through remote file system".

Still not sure about multiple commands in one line though, maybe @waldyrious @rubenvereecken @igorshubovych want to say something to this?

**Comment by waldyrious** [comment]
I am not sure I like the comma-separated command list (@rprieto), but I'll just comment on the phrasing: The command descriptions, for consistency, ought to be phrased as declarative statements, or instructions, e.g "do this". In this case, that would mean: "Temporary drop to local shell" --> "Temporarily drop to local shell" and "After dropping to local shell get back to `sftp` with:" --> "Get back to `sftp` after dropping to local shell". Does that make sense?

**Comment by igorshubovych** [comment]
I do not like comma in commands. There should be one command per description according to guidelines.

Also the page is already too long.

**Comment by slash3b** [comment]
Thank you for the comments. 
I removed "Temporarily drop to local shell" command for the list to be shorter, I left only `lls` and `ls`
 commands.

**Comment by waldyrious** [comment]
@slash3b shoulddn't there be at least one non-interactive version? The simplest use cases should be listed -- note that tldr doesn't aim to be a complete reference for the commands it covers, just provide a well-balanced intro/refresher.

**Comment by 85pando** [comment]
`sftp` is totally interactive. For non-interactive file copying to remote machines there is `scp` (and later `rsync`).

**Comment by waldyrious** [comment]
How about editing the main description to reflect that? Currently it seems to indicate quite the contrary :)

We could include a suggestion to use scp for single files or rsync for single directories (which I believe is a reasonable recommendation, but happy to stand corrected).

**Comment by waldyrious** [comment]
Sorry, clicked the wrong button

**Comment by slash3b** [comment]
@85pando, thank you.
@waldyrious, how about this description:

> Secure and interactive file transfer program. Performs all operations over an encrypted ssh transport.

**Comment by waldyrious** [comment]
@slash3b I'd include the expansion of "SFTP" intact in the description, and start with "interactive" to make that more evident. How about something along these lines?

> Interactive program to copy files between hosts using the Secure File Transfer Protocol over SSH.
> For non-interactive file transfers, see `scp` or `rsync`.

**Comment by slash3b** [comment]
Is it okay now ? I edited slightly description to make it shorter.

**Comment by rprieto** [comment]
:+1: on making it clear it's interactive. However unless I'm missing something obvious, I wouldn't mention `scp` or `rsync` since they won't help you get/put files on an FTP server.

**Comment by 85pando** [comment]
`sftp` and `scp` both only need an `sshd` on the server side. `rsync` needs to be installed on both sides of the communication. So `scp` actually has the same dependencies as `sftp`.

**Comment by rubenvereecken** [comment]
@rprieto maybe some confusion arose because of the name? There's `sftp` for file transfer (not ftp) over ssh and `ftps` (sometimes also called `sftp`) for FTP over SSL.

**Comment by waldyrious** [comment]
Let's just mention scp then? Also, are we ok with a 3-line description?

**Comment by rprieto** [comment]
> sftp and scp both only need an sshd on the server side

I didn't know that, thanks for explaining! Sounds good to me then :+1:

**Comment by ostera** [comment]
Many :+1:s! Seems like it's merging time :) before it gets stale. Thanks @slash3b and @85pando for your contributions!

**Comment by slash3b** [comment]
Thank you guys! 
Couldn't reply in time, my home machine is broken.
Will add some new command for sure.

**Pull request merged**